pom.xml

```diff
@@ -110,20 +103,7 @@
       <version>3.3.0</version>
       <type>maven-plugin</type>
     </dependency>
-
-    <dependency>
```

Comment:
We should also update the licenses in the "Settings->About" screen to reflect the change of fontawesome. We need to now reference and include the CC license.
 https://github.com/FortAwesome/Font-Awesome#license - Icons — CC BY 4.0 License